packages/block-library/src/navigation-menu/edit.js

```diff
@@ -151,7 +212,7 @@ function NavigationMenu( {
 				{ isRequesting && <><Spinner /> { __( 'Loading Navigation…' ) } </> }
 				{ pages &&
 					<InnerBlocks
-						template={ defaultMenuItems ? defaultMenuItems : null }
+						template={ blocksTemplate ? blocksTemplate : null }
```

Comment:
It looks like the default value of `blocksTemplate` is `null`, so probably no need for the ternary.

Reply:
No longer valid as we've ditched `template` all together 🎉